recipes/libtheora/meta.yaml

```diff
@@ -0,0 +1,45 @@
+package:
+  name: libtheora
+  version: 1.1.1
+
+source:
+  fn: libtheora-1.1.1.tar.bz2
+  url: http://downloads.xiph.org/releases/theora/libtheora-1.1.1.tar.bz2
```

Comment:
Would suggest using Jinja to template the version at least as in the example. Should make it easier to update (and less likely to release something under the wrong version).